test/integration/targets/aws_ec2_inventory/playbooks/create_inventory_config.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: 127.0.0.1
+  connection: local
+  gather_facts: no
+  tasks:
+    - name: write inventory config file
+      copy:
+        dest: ../test.aws_ec2.yml
+        content: |
```

Comment:
This should be a template! I'm all for the occasional use of copy with content but this is a little extreme!